R/plot_sankey.R

```diff
@@ -22,21 +22,23 @@
 #'
 #' @examples
 #' # Plot with equal weight and with company name
-#' plot_sankey(sankey_toy_data)
+#' plot_sankey(financial_toy_data)
 #'
 #' # Plot with best_case weight
-#' plot_sankey(sankey_toy_data, mode = "best_case")
+#' plot_sankey(financial_toy_data, mode = "best_case")
 plot_sankey <- function(data, with_company = TRUE, mode = c("equal_weight", "worst_case", "best_case", "main_activity")) {
   mode <- arg_match(mode)
 
-  limits <- c("Bank", if (with_company) "Company", NULL, "Tilt Sector", "PCTR risk category")
+  risk_category_var <- names(select(data, matches("_risk_category")))
+
+  limits <- c("Bank", if (with_company) "Company", NULL, "Tilt Sector", risk_category_var)
 
   p <- ggplot2::ggplot(
     data = data,
     aes(
       axis1 = .data$kg_id,
       axis3 = .data$tilt_sector,
-      axis4 = factor(.data$pctr_risk_category, levels = c("low", "medium", "high"))
+      axis4 = factor(.data[[risk_category_var]], levels = c("low", "medium", "high"))
```

Comment:
Well done. This makes the code more reusable and easier to maintian, as the "pctr_" detail is more likely to change than the "risk_category" concept.